Readability.js

```diff
@@ -613,6 +613,12 @@ Readability.prototype = {
       var node = this._doc.documentElement;
 
       while (node) {
+        // Remove hidden elements.
+        if (node.style && node.style.display === "none") {
```

Comment:
Maybe also test the presence of the `hidden` attribute?
https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute